src/main/java/pl/psnc/dei/service/TasksQueueService.java

```diff
@@ -0,0 +1,23 @@
+package pl.psnc.dei.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.psnc.dei.queue.TasksQueue;
+import pl.psnc.dei.queue.task.Task;
+
+@Service
+public class TasksQueueService {
+	
+	private static TasksQueue queue;
+
+	@Autowired
+	public TasksQueueService(TasksQueue tasksQueue) {
+		queue = tasksQueue;
```

Comment:
Is there going to be only one instance of this service? In case of more instances the static queue will be overwritten with the new one...am I right? I'm not sure how to describe it...but something is worrying me here...

Reply:
@Service annotation creates singleton bean by default, so there should by only 1 instance.

And yeah, it would be overwritten, i can change it so that i will not be static anymore.